Update CMEK FAQ (#16365)
cockroachdb/docs: cockroachcloud/cmek-faq.md

Yes, the {{ site.data.products.dedicated }} clusters are encrypted by default by the way of encrypting the cloud disks used to store the data, using cloud provider managed keys.

## What steps should I take before enabling CMEK for a cluster?

Contact your Cockroach Labs account team to enroll your organization in limited access for CMEK. CMEK can only be enabled on a private cluster, so your organization must also be enrolled in limited access for private clusters.

## Can we enable CMEK for an existing cluster that wasn't created as a private cluster?

CMEK can be enabled only on a private cluster. Contact your account team for advice about how to migrate or restore your existing cluster's data to a new private cluster.

## If we enable CMEK for a cluster that has been in use for some time, is the existing data encrypted at that time?

> {{ site.data.products.dedicated }} does not force encryption of previously-written data but instead relies on normal storage engine churn for desired encryption. That means the new key is used to encrypt newly-written data, while previously-written data remains unencrypted unless it’s rewritten.

Reviewer: ```suggestion
{{ site.data.products.dedicated }} does not force encryption of previously-written data but instead relies on normal storage engine churn for desired encryption. That means the new key is used to encrypt newly-written data, while previously-written data remains unencrypted unless it's rewritten.
```

> ## Can we enable CMEK when we add a new region to a cluster?

Reviewer 2: ```suggestion
## Can we enable CMEK for a new region when it's added to a CMEK-enabled cluster?
```

> Yes, when you add a new CMEK-enabled region to a cluster, you must also enable CMEK on its existing regions if it is not already enabled. Refer to [Activate CMEK for your {{ site.data.products.dedicated }} Cluster](/docs/cockroachcloud/managing-cmek.html#step-4-activate-cmek-for-your-cockroachdb-dedicated-cluster) and [Add a Region to a CMEK-enabled Cluster](/docs/cockroachcloud/managing-cmek.html#add-a-region-to-a-cmek-enabled-cluster).

Reviewer 2: ```suggestion
Yes, when you add a new region to a CMEK-enabled cluster, you can enable CMEK for that region. Refer to [Add a Region to a CMEK-enabled Cluster](/docs/cockroachcloud/managing-cmek.html#add-a-region-to-a-cmek-enabled-cluster).
```
